amy/extforms/forms.py

```diff
@@ -248,6 +252,12 @@ def validate_member_code(
 
         return errors
 
+    def clean_eventbrite_url(self):
+        """Check that entered URL includes 'eventbrite' in the domain."""
+        eventbrite_url = self.cleaned_data.get("eventbrite_url", "")
+        if eventbrite_url and "eventbrite" not in urlparse(eventbrite_url).netloc:
```

Comment:
If user provides `https://eventbrite.example.org/123123123` this fill validate. Do you think this is serious enough that we should whitelist eventbrite domain?

Also, you can use `urlparse(eventbrite_url).hostname`, as in some cases netloc contains more info than what we need.

Reply:
I found that Eventbrite have a lot of local domains as well as `eventbrite.com`, e.g. `eventbrite.co.uk`, `eventbrite.fr`, so it's harder to whitelist them.
As the link isn't clickable anywhere, I think this will be okay without further validation. But maybe for safety, I'll add something on the admin side that does a stricter check and shows a warning if the link doesn't look legitimate.